packages/sources/finage-test/src/endpoint/http/commodities.ts

```diff
@@ -0,0 +1,75 @@
+import { HttpTransport } from '@chainlink/external-adapter-framework/transports'
+import { PriceEndpoint, PriceEndpointParams } from '@chainlink/external-adapter-framework/adapter'
+import { SingleNumberResultResponse } from '@chainlink/external-adapter-framework/util'
+import { customSettings } from '../../config'
+
+export const inputParameters = {
+  base: {
+    aliases: ['from', 'coin'],
```

Comment:
The aliases in v2 are just `from` and `symbol`. But if coin needs to be in there for the `PriceEndpointParams` types is it possible to just add `symbol` or would that give you an error?